README.md

```diff
@@ -107,6 +110,8 @@ $ pip list --local | grep treon # should point to the source tree
 treon                0.1.2                /workspace/treon
 ```
 
+Please refer to the `Makefile` for supplementary development tasks, such as linting treon's source code.
```

Comment:
Can we please specify how to run the linter? People like me are not very familiar with makefiles :)

Also good to add note for future contributors to run linter before submitting PR.

Reply:
OK no problem 👍

Reply:
I am happy to merge the PR once you update the Readme.